inc/cache_enabler_disk.class.php

```diff
@@ -483,7 +483,7 @@ public static function create_settings_file( $settings ) {
         $new_settings_file = self::get_settings_file();
 
         // make directory if necessary
-        if ( ! wp_mkdir_p( dirname( $new_settings_file ) ) ) {
+        if ( ! self::mkdir_p( dirname( self::get_settings_file() ) ) ) {
```

Comment:
To understand, what is the reason behind `$new_settings_file` being replaced with `self::get_settings_file()`?

Reply:
Good catch, that was leftover from when I tried a few different approaches.